### ISSUE TITLE
camelCase false positive for words starting with 'C'

### ISSUE THREAD

**Issue by sseagull** [streetsidesoftware/vscode-spell-checker#627]
I primarily work in `js/ts` so this might be a language specific problem, but the spell checker seems to flag camel case words that start with the letter 'C' by not including the 'C' in the word.

Example: `masterColumns` flags a spelling error on the word `olumns`

<img width="391" alt="Screen Shot 2020-12-04 at 9 08 59 AM" src="https://user-images.githubusercontent.com/54673844/101173353-6c57c480-3610-11eb-94e7-c6107ae3b12a.png">

Note: this doesn't seem to happen with all camelCase words containing a `C` word, but it does happen often enough.

Maybe I missed a setting for this?

**Comment by Jason3S** [comment]
Do you have any regexp defined in the ignore list?

**Comment by sseagull** [comment]
Ah sorry, I should have included my configs.

```
"cSpell.enableFiletypes": [
    "css",
    "javascript",
    "javascriptreact",
    "markdown",
    "scss",
    "typescript",
    "typescriptreact"
  ],
  "cSpell.ignoreRegExpList": ["([a-z])+(ignore|rc|lint)"],
  "cSpell.ignoreWords": ["docgen"],
  "cSpell.ignorePaths": [
    "**/package-lock.json",
    "**/node_modules/**",
    "**/vscode-extension/**",
    "**/.git/objects/**",
    ".vscode"
  ],
```

**Comment by sseagull** [comment]
Oh wow, I think its _exactly_ that thats the problem. I was trying to skip config files like `.prettierrc`, guess my regex is faulty. Thanks for asking, I wouldn't have thought to check that setting. I guess Im good in terms of 'issue resolved' then.
